railties/lib/rails/generators/rails/app/templates/app/views/layouts/application.html.erb.tt

```diff
@@ -2,6 +2,7 @@
 <html>
   <head>
     <title><%= camelized %></title>
+    <meta name="viewport" content="width=device-width,initial-scale=1"/>
```

Comment:
Can you please remove the closing `/`? This matches what the modern `tag.meta` helper would produce.

```suggestion
    <meta name="viewport" content="width=device-width,initial-scale=1">
```
```